cmd/validate/image.go

```diff
@@ -393,6 +444,10 @@ func validateImageCmd(validate imageValidationFunc) *cobra.Command {
 		a RFC3339 formatted value, e.g. 2022-11-18T00:00:00Z.
 	`))
 
+	cmd.Flags().StringSliceVar(&data.extra, "extra", data.extra, hd.Doc(`
```

Comment:
The name "extra" feels not quite specific enough. Something like this perhaps:

```suggestion
	cmd.Flags().StringSliceVarP(&data.extraRuleData, "extra-rule-data", "e", data.extraRuleData, hd.Doc(`
```

Reply:
Changed it to your suggestion, but open to other ideas as well. I know we floated `override` or `override-rule-data` as well.